dll/win32/devmgr/lang/ro-RO.rc

```diff
@@ -247,14 +251,14 @@ END
 
 IDD_UNINSTALLDRIVER DIALOGEX 0, 0, 224, 121
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_POPUPWINDOW | WS_VISIBLE | WS_DLGFRAME
-CAPTION "Confirm Device Removal"
+CAPTION "Confirmați eliminarea dispozitivului"
 FONT 8, "MS Shell Dlg"
 BEGIN
     ICON "", IDC_DEVICON, 7, 7, 20, 20
     LTEXT "", IDC_DEVNAME, 40, 9, 174, 16, SS_NOPREFIX
-    LTEXT "Warning: You are about to uninstall this device from your system.", -1, 7, 37, 204, 50
+    LTEXT "Avertisment: Urmează să dezinstalați acest dispozitiv din sistemul dvs.", -1, 7, 37, 204, 50
     DEFPUSHBUTTON "OK", IDOK, 110, 100, 50, 14
-    PUSHBUTTON "Cancel", IDCANCEL, 166, 100, 50, 14
+    PUSHBUTTON "A&nulează", IDCANCEL, 166, 100, 50, 14
```

Comment:
en-US and other translations don't seem to have a mnemonic/access key for this button. If you want to add a mnemonic key here, it should be done separately from the translation.
```suggestion
    PUSHBUTTON "Anulează", IDCANCEL, 166, 100, 50, 14
```

Reply:
Ok, I changed it.